packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/perl/changeCall4.yml

```diff
@@ -0,0 +1,28 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change call
+  action: { name: clearAndSetSelection }
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: { type: functionCall }
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    my $var = 1;
+    $var = Other::Package::func();
+    $var = 2;
```

Comment:
Not worth re-recording, but these tests are a bit easier to read if they're smaller. Eg just the statement containing the function call